src/test/java/com/g4s8/ghman/data/PgUserITCase.java

```diff
@@ -29,14 +29,20 @@
  *  methods of {@link com.g4s8.ghman.data.PgUser} class. Do not forget about
  *  all possible exceptions.
  */
-@ExtendWith(DatabaseExtension.class)
 final class PgUserITCase {
 
+    /**
+     * Database.
+     * @checkstyle VisibilityModifierCheck (5 lines)
+     */
+    @RegisterExtension
+    static DatabaseExtension source = new DatabaseExtension();
```

Comment:
@olenagerasimova can we make that non-static? It will simply be instantiated for everytest instead of all the test of the class, it's safer I think.